srcpkgs/rofi-wayland/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.5+wayland1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config git flex glib-devel wayland-devel"
+makedepends="libglib-devel cairo-devel pango-devel libxkbcommon-devel
+ gdk-pixbuf-devel wayland-devel libmpdclient-devel libnl3-devel libxcb-devel
+ xcb-util-devel xcb-util-wm-devel xcb-util-cursor-devel
+ startup-notification-devel wayland-protocols alsa-lib-devel check-devel"
```

Comment:
check-devel should probably be in checkdepends

Reply:
check-devel is needed during compilation.